apps/prairielearn/python/zygote.py

```diff
@@ -68,6 +68,7 @@
 
 # Pre-load commonly used modules
 sys.path.insert(0, os.path.abspath("../question-servers/freeformPythonLib"))
```

Comment:
AFAICT this path simply doesn't exist anymore. Can we remove this?

Reply:
Removed in 2a955b47aaba80ca142730b9a9cdb229eddf92a8.